src/bootstrap/mk/Makefile.in

```diff
@@ -108,11 +108,21 @@ ci-msvc: ci-msvc-py ci-msvc-ps1
 
 ## MingW native builders
 
+# Set of tests that should represent half of the time of the test suite.
+# Used to split tests across multiple CI runners.
+TESTS_IN_MINGW_2 := \
+	tests/ui \
+	tests/rustdoc \
+	tests/run-make \
+	tests/rustdoc-ui \
+	tests/ui-fulldeps \
+	tests/rustdoc-json
+
 # test both x and bootstrap entrypoints
 ci-mingw-x:
-	$(Q)$(CFG_SRC_DIR)/x test --stage 2 tidy
+	$(Q)$(CFG_SRC_DIR)/x test --stage 2 $(TESTS_IN_MINGW_2)
 ci-mingw-bootstrap:
-	$(Q)$(BOOTSTRAP) test --stage 2 --skip tidy
+	$(Q)$(BOOTSTRAP) test --stage 2 $(TESTS_IN_MINGW_2:%=--skip %)
```

Comment:
What does this magic do? Prefix `--skip` before each line? We should probably wait for https://github.com/rust-lang/rust/pull/132979 with this, btw.

Reply:
Yeah, makes sense to wait. The best would be if they support regex. It would be nice to do `tests/*`.